Resources/Prototypes/Body/Organs/diona.yml

```diff
@@ -29,8 +29,11 @@
   id: OrganDionaBrain
   parent: [BaseDionaOrgan, OrganHumanBrain]
   name: brain
-  description: "The source of incredible, unending intelligence. Honk."
+  description: "The central hub of a diona's pseudo-neurological activity, its root-like tendrils search for its former body."
```

Comment:
it’s

Reply:
erm akshually the possessive form of it does not contain a '
![image](https://github.com/user-attachments/assets/6294d9e9-6c94-4286-8030-217614b451d5)